src/XGBoost.jl

```diff
@@ -9,7 +9,12 @@ export DMatrix, Booster
 export xgboost, predict, save, nfold_cv, slice, get_info, set_info, dump_model, importance
 export rabit_init, rabit_finalize, rabit_is_distributed, rabit_get_rank, rabit_get_world_size, rabit_get_version_number
 
+global const build_version = "0.82"
```

Comment:
can this be somehow generated in the build.jl, so that if we upgrade the dep, this upgrades as well? not a big deal.

Reply:
It's not generated right now, but you could put it in there if you want.